db/migrate/20230605171154_create_hikes.rb

```diff
@@ -0,0 +1,18 @@
+class CreateHikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :hikes do |t|
+      t.string :name
+      t.string :length
+      t.text :description
+      t.string :city
+      t.string :region
+      t.string :country
+      t.string :lat
+      t.string :lon
```

Comment:
If you plan to use the Ruby Geocoder library, these will be come conventional stored as `:latitude` and `:longitude` and their date type needs to either be `float`, or `decimal` but I personal prefer `decimal`